tests/test_flow.ts

```diff
@@ -1777,7 +1778,7 @@ describe('flows', () => {
     const { children, job } = tree;
     const parentState = await job.getState();
 
-    //expect(parentState).to.be.eql('waiting-children');
+    expect(parentState).to.be.eql('waiting-children');
```

Comment:
after the fix we in bullmq this is not failing
before I though this is a test bug that we finished running all jobs before we reached this